did-pkh/src/lib.rs

```diff
@@ -475,6 +524,11 @@ mod tests {
             "eth",
             "did:pkh:eth:0x2fbf1be19d90a29aea9363f4ef0b6bf1c4ff0758",
         );
+        test_generate(
+            secp256k1_pk.clone(),
+            "celo",
+            "did:pkh:celo:0xa0ae58da58dfa46fa55c3b86545e7065f90ff011",
```

Comment:
```suggestion
            "did:pkh:celo:0x2fbf1be19d90a29aea9363f4ef0b6bf1c4ff0758",
```
The address here should be the same as the `eth` one above, since it is using the same keypair.
This should fix the CI failure https://github.com/spruceid/ssi/runs/3020690586#step:5:511

Reply:
Got it! The weird thing is that it's not failing at my machine.

Reply:
Maybe the test is not running because it depends on the non-default secp256k1 feature.
This should make it run (in the repo root directory):
`cargo test -p did-pkh --features=secp256k1,keccak-hash`